### ISSUE TITLE
Please review new library API specification

### ISSUE THREAD

**Issue by DrDaveD** [csf-ngs/hinkskalle#1]
Please review the newly written specification for the singularity library protocol at https://github.com/singularityhub/library-api based on your knowledge of the API, and consider volunteering to be a maintainer as described in singularityhub/library-api#1.

Thanks!

**Comment by h3kker** [comment]
https://github.com/sylabs/scs-library-client/blob/master/api/server/openapi.yml

**Comment by DrDaveD** [comment]
Yes, there's a link to that from the web page, but it's only a subset of the API and only for v1.  It's also in a repository controlled by one company, and the point of having the standard organization is to have changes to the API maintained by the community.
